### PR TITLE
Enhanced German translation

### PR DESCRIPTION
**Pull request by lupushortus**
Added new strings
Utilize strings in GOFrame, where not yet done

### CODE REVIEW

File: src/grandorgue/GOFrame.cpp [hunk 1]
```diff
@@ -222,7 +222,7 @@ GOFrame::GOFrame(
     wxITEM_NORMAL);
   m_file_menu->AppendSeparator();
   m_file_menu->Append(
-    ID_SETTINGS, wxT("&Settings..."), wxEmptyString, wxITEM_NORMAL);
+    ID_SETTINGS, _("&Settings..."), wxEmptyString, wxITEM_NORMAL);
```
Comment (oleg68) [inline src/grandorgue/GOFrame.cpp:225]: `Settings` was not translated by design: it allows to user find this menu item whenever the corrent language is wrong.

File: src/grandorgue/GOFrame.cpp [hunk 2]
```diff
@@ -720,7 +720,7 @@ void GOFrame::UpdateTemperamentMenu() {
       }
     }
     wxMenuItem *e = menu->Append(
-      ID_TEMPERAMENT_0 + i, t.GetTitle(), wxEmptyString, wxITEM_CHECK);
+      ID_TEMPERAMENT_0 + i, wxGetTranslation(t.GetTitle()), wxEmptyString, wxITEM_CHECK);
```
Comment (kerkovits) [inline src/grandorgue/GOFrame.cpp:674]: This is perhaps not the best solution. In older versions of GrandOrgue, `t.GetTitle()` already returned translated temperament names. Please see also https://github.com/GrandOrgue/grandorgue/issues/1104

**Comment by oleg68** [comment]
@Marco Wolfgarten This PR is about the translation. Please do not push different unrelated changes with a single PR. Create a separate PR for crescendo

**Comment by lupushortus** [comment]
@oleg68: Now only translation file is subject to PR.

**Review by oleg68 — APPROVED** [review]

**Comment by oleg68** [comment]
@Mikelectric could you check this translation?

**Pull request merged**